lib/src/switch/Switch.tsx

```diff
@@ -51,73 +48,59 @@ const DxcSwitch = ({
     if (checked === undefined) {
```

Comment:
This code can be stretched again. Proposal:
```
  const handlerSwitchChange = () => {
    checked ?? setInnerChecked((innerChecked) => !innerChecked);
    onChange?.(checked ? !checked : !innerChecked);
  };
```

Reply:
Done